bridge/src/mock.rs

```diff
@@ -325,6 +345,77 @@ impl ConcrateSygmaAsset {
 	}
 }
 
+pub struct SygmaDecimalConverter;
+impl DecimalConverter for SygmaDecimalConverter {
```

Comment:
We can pass decimal pairs to the converter by adding a generic parameter

```suggestion
pub struct SygmaDecimalConverter<DecimalPairs>;
impl<DecimalPairs: Get<Vec<(AssetId, u8)>>> DecimalConverter for SygmaDecimalConverter<DecimalPairs> {
```

Reply:
I understand the logic here, but I got error when run testing, saying that in `pub struct SygmaDecimalConverter<DecimalPairs>; DecimalPairs is an unused parameter.` since there is no field in struct SygmaDecimalConverter, I think it makes sense, but in this case how to initialize it with and pass in `AssetDecimalPairs`?